data-prepper-core/src/main/java/org/opensearch/dataprepper/peerforwarder/PeerForwarderConfiguration.java

```diff
@@ -84,6 +85,7 @@ public PeerForwarderConfiguration (
             @JsonProperty("static_endpoints") final List<String> staticEndpoints,
             @JsonProperty("client_thread_count") final Integer clientThreadCount,
             @JsonProperty("batch_size") final Integer batchSize,
+            @JsonProperty("batch_delay") final Integer batchDelay,
```

Comment:
Should we call this `delay` to remain inline with the pipeline buffer terminology?

Reply:
I'm not sure if making it `delay` would provide any context to it. We can revisit this once we have some default values for `batch_size`, `batch_delay` and `request_timeout`. We also need to update documentation.